### DISCUSSION TITLE
audio enhancement features

### DISCUSSION THREAD

**Discussion by IJustItay** [pear-devs/pear-desktop#943] [Ideas]
Because the lack of hi res audio and spatial sound in youtube music I think it would be cool if you will bring audio enhancement like cider project did. for more information: https://cider.sh/

**Comment by donneeee** [comment]
i second this, as there is another youtube music app somewhere out there that does work, but doesn't look or run as smooth as this one; it's just missing spatial audio options (and for me the only real thing i miss from the other app i was using).

**Reply by Mendungo** [reply]
Which app is that? does it download the songs in better quality formats?

**Comment by Malakasskatakys** [comment]
maybe an equalizer and pitch shift
